modyn/config/schema/pipeline/trigger/performance/criterion.py

```diff
@@ -0,0 +1,99 @@
+from typing import Annotated, Literal
+
+from pydantic import Field
+
+from modyn.config.schema.base_model import ModynBaseModel
+
+# -------------------------------------------------------------------------------------------------------------------- #
+#                                                 PerformanceCriterion                                                 #
+# -------------------------------------------------------------------------------------------------------------------- #
+# uses the evaluation results directory to derive a triggering decision
```

Comment:
directory?

Reply:
It doesn't make sense; probably a code pilot artifact😅